img/feature-auroc.eps

```diff
@@ -0,0 +1,1239 @@
+/cairo_store_point { /cairo_point_y exch def /cairo_point_x exch def } def
+/Tj { show currentpoint cairo_store_point } bind def
+/TJ {
+  {
+    dup
+    type /stringtype eq
+    { show } { -0.001 mul 0 cairo_font_matrix dtransform rmoveto } ifelse
+  } forall
+  currentpoint cairo_store_point
+} bind def
+/cairo_selectfont { cairo_font_matrix aload pop pop pop 0 0 6 array astore
+    cairo_font exch selectfont cairo_point_x cairo_point_y moveto } bind def
+/Tf { pop /cairo_font exch def /cairo_font_matrix where
+      { pop cairo_selectfont } if } bind def
+/Td { matrix translate cairo_font_matrix matrix concatmatrix dup
+      /cairo_font_matrix exch def dup 4 get exch 5 get cairo_store_point
+      /cairo_font where { pop cairo_selectfont } if } bind def
+/Tm { 2 copy 8 2 roll 6 array astore /cairo_font_matrix exch def
+      cairo_store_point /cairo_font where { pop cairo_selectfont } if } bind def
+/g { setgray } bind def
+/rg { setrgbcolor } bind def
+/d1 { setcachedevice } bind def
+/cairo_data_source {
+  CairoDataIndex CairoData length lt
+    { CairoData CairoDataIndex get /CairoDataIndex CairoDataIndex 1 add def }
+    { () } ifelse
+} def
+/cairo_flush_ascii85_file { cairo_ascii85_file status { cairo_ascii85_file flushfile } if } def
+/cairo_image { image cairo_flush_ascii85_file } def
+/cairo_imagemask { imagemask cairo_flush_ascii85_file } def
+%%EndProlog
+%%BeginSetup
+%%BeginResource: font Helvetica
+11 dict begin
+/FontType 42 def
+/FontName /Helvetica def
+/PaintType 0 def
+/FontMatrix [ 1 0 0 1 0 0 ] def
+/FontBBox [ 0 0 0 0 ] def
+/Encoding 256 array def
+0 1 255 { Encoding exch /.notdef put } for
+Encoding 32 /space put
+Encoding 46 /period put
+Encoding 47 /slash put
+Encoding 48 /zero put
+Encoding 49 /one put
+Encoding 53 /five put
+Encoding 54 /six put
+Encoding 55 /seven put
+Encoding 56 /eight put
+Encoding 57 /nine put
+Encoding 65 /A put
+Encoding 67 /C put
+Encoding 74 /J put
+Encoding 79 /O put
+Encoding 80 /P put
+Encoding 82 /R put
+Encoding 84 /T put
+Encoding 85 /U put
+Encoding 97 /a put
+Encoding 99 /c put
+Encoding 100 /d put
+Encoding 101 /e put
+Encoding 102 /f put
+Encoding 103 /g put
+Encoding 104 /h put
+Encoding 105 /i put
+Encoding 107 /k put
+Encoding 108 /l put
+Encoding 109 /m put
+Encoding 110 /n put
+Encoding 111 /o put
+Encoding 112 /p put
+Encoding 114 /r put
+Encoding 115 /s put
+Encoding 116 /t put
+Encoding 117 /u put
+Encoding 119 /w put
+Encoding 120 /x put
+/CharStrings 39 dict dup begin
+/.notdef 0 def
+/P 1 def
+/r 2 def
+/e 3 def
+/f 4 def
+/n 5 def
+/t 6 def
+/i 7 def
+/a 8 def
+/l 9 def
+/space 10 def
+/c 11 def
+/h 12 def
+/m 13 def
+/d 14 def
+/x 15 def
+/A 16 def
+/slash 17 def
+/J 18 def
+/R 19 def
+/s 20 def
+/o 21 def
+/u 22 def
+/w 23 def
+/k 24 def
+/zero 25 def
+/period 26 def
+/five 27 def
+/six 28 def
+/seven 29 def
+/eight 30 def
+/nine 31 def
+/one 32 def
+/U 33 def
+/O 34 def
+/C 35 def
+/T 36 def
+/g 37 def
+/p 38 def
+end readonly def
+/sfnts [
```

Comment:
For `img/feature-auroc.eps`, it might be easiest to just remove transparency from points altogether. Their fill can be adjusted to keep the same colors if desired. There's a single overplotting, but we'd still see some of the background point. Dashed vline should go behind points if easy.